spec/support/disable_animation.rb

```diff
@@ -25,6 +25,8 @@ def call(env)
     private
 
     def html?
+      return false unless @headers['Content-Type']
```

Comment:
Why do we need to change here? 🤔

Reply:
Because Websockets generates requests with empty headers:
![image](https://user-images.githubusercontent.com/77609814/126944049-af3cc18b-a507-4a73-abb6-89d35e81deb7.png)

So without this line, all system tests navigating to the home page (that has web sockets connections) will return a nill error:
![image](https://user-images.githubusercontent.com/77609814/126944276-93dff4c5-e759-4a32-8f79-8e9387db2b90.png)

I'm not sure though if it's an issue from Rails-template or a issue on my web sockets configuration? 👀

Reply:
For this filtering may be we can use Ruby's safe navigation operator `&` to do it in one line
```
@headers['Content-Type']&.include?('html')
```

And maybe up we can fix it in the template also as it generates an error if the header is empty

Reply:
on it :) Thanks!

Reply:
Fixed & I will implement it in rails-template now (bug declared [here](https://github.com/nimblehq/rails-templates/issues/297))